docs/sections/user_guide/cli/mode_rocoto.rst

```diff
@@ -75,4 +228,100 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use a Rocoto XML file ``rocoto.xml`` with the following content:
+
+.. code:: XML
+
+  <?xml version='1.0' encoding='utf-8'?>
+  <!DOCTYPE workflow [
+    <!ENTITY ACCOUNT "myaccount">
+    <!ENTITY FOO "test.log">
+  ]>
+  <workflow realtime="False" scheduler="slurm">
+    <cycledef group="howdy">202209290000 202209300000 06:00:00</cycledef>
+    <log>/some/path/to/&FOO;</log>
+    <task name="hello" cycledefs="howdy">
+      <account>&ACCOUNT;</account>
+      <nodes>1:ppn=1</nodes>
+      <walltime>00:01:00</walltime>
+      <command>echo hello $person</command>
+      <jobname>hello</jobname>
+      <envar>
+        <name>person</name>
+        <value>siri</value>
+      </envar>
+    </task>
+  </workflow>
+
+
+* To validate an XML from ``stdin``:
+
+  .. code:: sh
+
+    $ cat rocoto.xml | uw rocoto validate
+    [2024-01-02T14:18:46]     INFO 0 Rocoto validation errors found
+
+* To validate an XML from file ``rocoto.xml``:
+
+  .. code:: sh
+
+    $ uw rocoto validate --input-file rocoto.xml
+    [2024-01-02T14:18:46]     INFO 0 Rocoto validation errors found
+
+* When the XML is invalid:
+
+  In this example, the ``<command>`` line was removed from the XML.
+
+  .. code:: sh
+
+    $ uw rocoto validate --input-file rocoto.xml
+    [2024-01-02T14:26:09]    ERROR 3 Rocoto validation errors found
+    [2024-01-02T14:26:09]    ERROR <string>:9:0:ERROR:RELAXNGV:RELAXNG_ERR_NOELEM: Expecting an element command, got nothing
+    [2024-01-02T14:26:09]    ERROR <string>:9:0:ERROR:RELAXNGV:RELAXNG_ERR_INTERSEQ: Invalid sequence in interleave
+    [2024-01-02T14:26:09]    ERROR <string>:9:0:ERROR:RELAXNGV:RELAXNG_ERR_CONTENTVALID: Element task failed to validate content
+
+  To decode this type of output, it is easiest to interpret it from the bottom up. It says:
+
+  * The task starting at Line 9 has invalid content.
+  * There was an invalid sequence.
+  * It was expecting a ``command`` element, but there wasn't one.
```

Comment:
Would
> It was expecting a `<command>` element, but there wasn't one.

be helpful, i.e. to tie the word `command` in the error statement `Expecting an element command, got nothing` to the explicitly bit of missing XML syntax, `<command>`? Someone experienced with XML may recognize that "an element command" refers to `<command>` in XML syntax, but "Expecting an element command" could mean a lot of things to the uninitiated. A tiny hint in our wording might help.

Reply:
Totally agree.